# PRI_VICON_COV: 0.05

# PRIOR_U_F_COV: 10.0
# PRIOR_U_M1_COV: 0.1
# PRIOR_U_M2_COV: 0.1
# PRIOR_U_M3_COV: 0.1

# CONTROL_P_COV: 0.03
# CONTROL_P_FINAL_COV: 0.005
# CONTROL_R_COV: 0.01

# DYNAMIC_P_COV: 0.001

# OPT_LENS_TRAJ: 20

# SIM_STEPS: 30000

PRI_VICON_COV:     0.04
PRI_VICON_VEL_COV: 0.0025

# PRI_VICON_COV:     0.000001
# PRI_VICON_VEL_COV: 0.000025

PRIOR_U_F_COV:  5.0
PRIOR_U_M1_COV: 0.1
PRIOR_U_M2_COV: 0.1
PRIOR_U_M3_COV: 0.1

CONTROL_P_COV_X:       0.0025
CONTROL_P_FINAL_COV_X: 0.0001
CONTROL_P_COV_Y:       0.0025
CONTROL_P_FINAL_COV_Y: 0.0001
CONTROL_P_COV_Z:       0.0025
CONTROL_P_FINAL_COV_Z: 0.0001

CONTROL_R_COV: 1000
CONTROL_V_COV: 100.0
CONTROL_O_COV: 100.0

DYNAMIC_P_COV: 0.0001

OPT_LENS_TRAJ: 20

SIM_STEPS: 100000

INPUT_JERK_T: 1
INPUT_JERK_M: 1